library/centos

```diff
@@ -7,11 +7,11 @@ Tags: latest, centos7, 7
 GitFetch: refs/heads/CentOS-7
 GitCommit: 45cccc8f40650e860852cfe9054eefa5a4c0dc54
```

Comment:
Was this one (and possibly i386 below) supposed to be updated as well?

Reply:
bah, yes.